TestSelectSSHKeys fails locally

I have `id_rsa` and `id_rsa.pub` in my `~/.ssh/` directory, but I expected the test to be isolated from that:

```
go test -run '^TestSelectSSHKeys$' ./pkg/cmd/codespace

--- FAIL: TestSelectSSHKeys (0.05s)
    /Users/mislav/p/gh-cli/pkg/cmd/codespace/ssh_test.go:261: Want selectSSHKeys result to be &{codespaces.auto.pub codespaces.auto}, got &{id_rsa.pub id_rsa}
    /Users/mislav/p/gh-cli/pkg/cmd/codespace/ssh_test.go:261: Want selectSSHKeys result to be &{codespaces.auto.pub codespaces.auto}, got &{id_rsa.pub id_rsa}
FAIL
```

Ref. #6304 /cc @cmbrose

--- a/pkg/cmd/codespace/ssh_test.go
+++ b/pkg/cmd/codespace/ssh_test.go
@@ -212,21 +212,21 @@ func TestSelectSSHKeys(t *testing.T) {
 			f.Close()
 		}
 
-		if tt.sshConfigKeys != nil {
-			configPath := filepath.Join(sshDir, "test-config")
+		configPath := filepath.Join(sshDir, "test-config")
 
-			configContent := ""
-			for _, key := range tt.sshConfigKeys {
-				configContent += fmt.Sprintf("IdentityFile %s\n", filepath.Join(sshDir, key))
-			}
+		// Seed the config with a non-existent key so that the default config won't apply
+		configContent := "IdentityFile dummy\n"
 
-			err := os.WriteFile(configPath, []byte(configContent), 0666)
-			if err != nil {
-				t.Fatalf("could not write test config %v", err)
-			}
-
-			tt.sshArgs = append(tt.sshArgs, "-F", configPath)
+		for _, key := range tt.sshConfigKeys {
+			configContent += fmt.Sprintf("IdentityFile %s\n", filepath.Join(sshDir, key))
 		}
+
+		err := os.WriteFile(configPath, []byte(configContent), 0666)
+		if err != nil {
+			t.Fatalf("could not write test config %v", err)
+		}
+
+		tt.sshArgs = append([]string{"-F", configPath}, tt.sshArgs...)
 
 		gotKeyPair, gotShouldAddArg, err := selectSSHKeys(context.Background(), sshContext, tt.sshArgs, sshOptions{profile: tt.profileOpt})
 

